src/client/components/editor/help.js

```diff
@@ -76,11 +76,14 @@ class Help extends React.Component {
     let showHelp = this.data.showHelp;
     let editable = this.props.document.editable();
     let bus = this.props.bus;
-    let cy = this.props.controller.data.cy;
+    let controller = this.props.controller;
+    let cy = controller.data.cy;
```

Comment:
It would be preferable for `cy` to be sent as a prop to `Help` rather than knowing about controller internals

Reply:
If I pass cy as props I run into some react rendering issues.  I can try to fiddle around some more though.